bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -35,17 +35,10 @@ spec:
           #!/bin/bash
           set -euo pipefail
           export SYSTEMD_IGNORE_CHROOT=yes
+
           # Check to see if ovs is provided by the node:
-          if systemctl status ovsdb-server; then
+          if [[ -f '/host/usr/local/bin/configure-ovs.sh' ]]; then
             echo "openvswitch is running in systemd"
-            # In some very strange corner cases, the owner for /run/openvswitch
```

Comment:
just add this back, because we know now the root cause of the e2e-upgrade job failure is not this. We can remove it later once @squeed is back.